### ISSUE TITLE
It doesn't work :S

### ISSUE THREAD

**Issue by dotku** [davidshimjs/qrcodejs#77]
Did you test it?!

**Comment by vdanelia** [comment]
Yes I tested and it works, but I needed with some more options :)

**Comment by mrmikardo** [comment]
I also could not get it to work - @vdanelia, could you specify what you did to get an output?

**Comment by vdanelia** [comment]
give me your mail and i'll send the working code with sets of options

**Comment by mrmikardo** [comment]
Hi Vakho,

It's jackwestmore@gmail.com

Cheers
Jack
